packages/wp-story-editor/src/index.js

```diff
@@ -35,8 +35,9 @@ import { initializeTracking } from '@web-stories-wp/tracking';
 /**
  * Internal dependencies
  */
-import './style.css'; // This way the general editor styles are loaded before all the component styles.
+import * as apiCallbacks from './api';
 import { PostPublishDialog } from './components';
+import './style.css'; // This way the general editor styles are loaded before all the component styles.
```

Comment:
Why was the order of imports changed here? Now the comment is not accurate anymore.

Aside: it might not be true anyway since some other components are imported way earlier now, so to be actually load those styles earlier this import would probably need to be moved to line ~25, before `External dependencies`

Reply:
I messed up in resolving a merge conflict. 😬